front/src/modules/ui/dialog/components/Dialog.tsx

```diff
@@ -86,6 +92,38 @@ export function Dialog({
     closed: { y: '50vh' },
   };
 
+  useHotkeys(
```

Comment:
Why not using useScopedHotkey ?

Reply:
We could use a DialogHotkeyScope that is set by usePreviousHotkeyScope and then we go back to previous scope on close.

Reply:
@lucasbordeau done